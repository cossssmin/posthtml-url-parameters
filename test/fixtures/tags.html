<a href="https://example.com">Test</a>
<link rel="stylesheet" href="https://example.com/style.css">
<module href="https://example.com/header.html">
