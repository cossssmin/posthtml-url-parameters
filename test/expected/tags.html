<a href="https://example.com?foo=bar">Test</a>
<link rel="stylesheet" href="https://example.com/style.css?foo=bar">
<module href="https://example.com/header.html">
</module>
